templates/flavors/azure-cni-v1/patches/azure-cluster-template.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
```

Comment:
why is this file called `azure-cluster-template.yaml`? It's an `AzureCluster` resource, not `AzureClusterTemplate` (`AzureClusterTemplate` also exists in the context of ClusterClass which could make this confusing)

Reply:
You are right.
It has now been updated to `azure-cluster.yaml`. 
I used the suffix `-template` since it was the template I was basing final manifest off of.